clients/web/src/routes/management/index.svelte

```diff
@@ -9,7 +13,7 @@
 
     export const load: Load = ({session}) => {
         if (session.user) {
-            if (session.user.orgTeams.some((s:number) => s === MLEDB_ADMIN)) {
+            if (session.user.orgTeams.some(s => s === MLEDB_ADMIN)) {
```

Comment:
Why remove the type declaration?

Reply:
It isn't needed since orgTeams is defined as number[], it already knows it's a number.